modules/exploits/multi/http/uptime_code_exec.rb

```diff
@@ -0,0 +1,394 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+class Metasploit4 < Msf::Exploit::Remote
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::PhpEXE
+
+  def initialize(info = {})
+    super(update_info(info,
+                      'Name' => 'Uptime Version 7.4.0 / 7.5.0 Upload and Exec file',
+                      'Description' => '
+                          This module exploits a vulnerability found in Uptime
+                          version 7.4.0 and 7.5.0 By supplying a specially
+                          crafted request is possible to upload file and execute
+                          arbitrary commands through privilege escalation.
+                      ',
+                      'License'        => MSF_LICENSE,
+                      'Author'         => [
+                        'Ewerson Guimaraes(Crash) <crash[at]dclabs.com.br>',
+                        'Gjoko Krstic(LiquidWorm) <gjoko[at]zeroscience.mk>'
+                      ],
+                      'References' => [
+                        ['CVE', ''],
+                        ['OSVDB', ''],
+                        ['US-CERT-VU', '']
```

Comment:
Remove these references for now. Once you obtained the CVE and other IDs, you can include them later.